package.json

```diff
@@ -253,7 +253,7 @@
         "svgo-loader": "^2.1.0",
         "uglifyjs-webpack-plugin": "^1.2.7",
         "url-loader": "^1.0.1",
-        "webpack": "^4.16.5",
+        "webpack": "4.16.5",
```

Comment:
Do you see any compatibility issues in future version? Or what is the reasoning to enforce exact version here?

Reply:
Yes, we experienced a problem trying to deploy with the webpack 4.20. It seems like the webpack-cli should be updated to the 3.1.1 version as well, let me just remove this changes.